src/olympia/amo/tests/ui/conftest.py

```diff
@@ -0,0 +1,64 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
In Web QA projects we include the license in every file. I can remove it though if this repo has a different policy.